src/proxy/NearCachedMapProxy.ts

```diff
@@ -194,7 +194,14 @@ export class NearCachedMapProxy<K, V> extends MapProxy<K, V> {
     }
 
     protected removeAllInternal(predicate: Predicate): Promise<void> {
```

Comment:
Can you change removeAllInternal to be as follows logicwise:

```java
try {
    super.removeAllInternal(predicate);
} finally {
    nearCache.clear();
}
```

Reply:
https://github.com/hazelcast/hazelcast/blob/14ef0dbefa4a6fa00a50a8873aeea02ec6c3f5e4/hazelcast/src/main/java/com/hazelcast/client/map/impl/nearcache/NearCachedClientMapProxy.java#L204

Reply:
@fatihozer0 It should be as follows:


```js
try {
    return super.removeAllInternal(predicate);
} finally {
    this.nearCache.clear();
}
```